### DISCUSSION TITLE
Traffic Data - OSM import

### DISCUSSION THREAD

**Discussion by ioannides-dB** [Universite-Gustave-Eiffel/NoiseModelling#314] [Q&A] [answered]
Hi all,

Importing osm file generates a roads table where specific traffic data is quoted for the various roads of the area of interest. I understand that this is not provided by the osm file as I couldn't find  such info attached to it. Thus I assume it is generated during the import part? Is it estimated or is there a database that NoiseModelling is retrieving data from? If estimated is there a methodology to reference? Or if from a database, is there a way to find out the source/author for referencing purposes?

Attached is the osm file if helpful. 
[map.zip](https://github.com/Ifsttar/NoiseModelling/files/6005654/map.zip)

Thanx!
Stefanos

**Comment by pierromond** [comment, marked as answer]
To get default value for traffic data, we used this document : [Good Practice Guide for Strategic Noise Mapping and the Production of Associated Data on Noise Exposure Version 2 13th January 2006 ](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwi-8rf3z_XuAhWL8eAKHXtOBoEQFjAEegQIAxAD&url=http%3A%2F%2Fsicaweb.cedex.es%2Fdocs%2Fdocumentacion%2FGood-Practice-Guide-for-Strategic-Noise-Mapping.pdf&usg=AOvVaw3a_oh8rU2DbSjuESLYxBAw)

#303 try to improve this way to do. You can share with them if you want !

**Reply by ioannides-dB** [reply]
Hi @pierromond, I understand that you have used the 2.5 Toolkit from Good practice guide (as above) during the OSM import. This produces traffic flow per period (day-evening-night). See attached image and footnote. The CNOSSOS-EU formula though uses a yearly average per hour flow. Am I missing something? Are you using some other method to produce per hour figures?

Thanx!

![image](https://user-images.githubusercontent.com/78847376/115268339-d77ae100-a142-11eb-90b5-0bcfada7dfc7.png)

**Reply by pierromond** [reply]
That's a very good point ! I never realized about this footprint... But it seems to be very low values... Maybe not, it depends on the road defintion. We should open an issue on it !

**Comment by pierromond** [comment]
I change the document and WPS Block to be more clear on this point.

- https://github.com/Ifsttar/NoiseModelling/commit/d0c69f0b744fb888f2e18a0463313f1eabb7f331
- https://github.com/Ifsttar/NoiseModelling/commit/1ae83644c62d954211c94a77af34b1c17005360f
